remote-query/remote-query-server/src/main/java/org/infinispan/query/remote/impl/ProtobufMetadataManagerImpl.java

```diff
@@ -95,16 +95,24 @@ void start() {
             getProtobufMetadataCacheConfig(globalConfiguration).build(),
             EnumSet.of(InternalCacheRegistry.Flag.USER, InternalCacheRegistry.Flag.PROTECTED, InternalCacheRegistry.Flag.PERSISTENT));
 
-      processSerializationContextInitializer(ServiceFinder.load(SerializationContextInitializer.class, globalConfiguration.classLoader()));
-      processSerializationContextInitializer(globalConfiguration.serialization().contextInitializers());
+      Collection<SerializationContextInitializer> initializers = globalConfiguration.serialization().contextInitializers();
+      if (initializers == null || initializers.isEmpty()) {
+         initializers = ServiceFinder.load(SerializationContextInitializer.class, globalConfiguration.classLoader());
+      }
+      processSerializationContextInitializer(initializers);
       processProtostreamSerializationContextInitializers(globalConfiguration.classLoader());
    }
 
+   /**
+    * @deprecated Since 10.
+    */
+   @Deprecated
    private void processProtostreamSerializationContextInitializers(ClassLoader classLoader) {
```

Comment:
This was deprecated in 10.x, I think we can remove this now along with `ProtostreamSerializationContextInitializer`

Reply:
Done